src/sqlfluff/core/rules/functional/segments.py

```diff
@@ -88,11 +88,19 @@ def raw_segments(self) -> "Segments":
             raw_segments_list.extend(s.raw_segments)
         return Segments(*raw_segments_list, templated_file=self.templated_file)
 
+    def recursive_crawl_all(self) -> "Segments":  # pragma: no cover
```

Comment:
Do none of the L042 test cases cover this so do we need the pragma no cover?

Reply:
They don't use it. I wouldn't have had to add it, but it seemed good to keep `Segments` API somewhat aligned with the underlying `BaseSegment`.